src/components/Datatable/ToolsTabItem/ToolsTabItem.types.ts

```diff
@@ -0,0 +1,12 @@
+import { ButtonProps } from '../../Button/Button.types';
+import { ToolsTabItemStates } from './ToolsTabItem.enum';
+
+export type ItemStates = typeof ToolsTabItemStates[keyof typeof ToolsTabItemStates];
+
+export interface ToolsTabItemProps extends ButtonProps {
+  itemState?: ItemStates;
+  activateTool: React.MouseEventHandler;
```

Comment:
It's better to start handlers with `on` prefix to distinguish it with data props:
- `onToolActivate`
- `onToolDeactivate`